<launch>
    <arg name="start_signal_path" default = "$(find bayesopt_ros)/media/start_signal.txt"/>
    <arg name="cost_path" default = "$(find bayesopt_ros)/media/cost.txt"/>
    <arg name="optimization_history_path" default = "$(find bayesopt_ros)/media/optimization_history.csv"/>  

    <param name="start_signal" type="string" value="$(arg start_signal_path)" />
    <param name="cost" type="string" value="$(arg cost_path)" />
    <param name="optimization_history" type="string" value="$(arg optimization_history_path)" />

    <param name="name" value="slam_bayesopt"/>
    <rosparam file="$(find bayesopt_ros)/config/bayesParams.yaml" command="load" /> 
    <node name="slam_bayesopt" pkg = "bayesopt_ros" type = "slam_bayesopt" output="screen">

    </node>
</launch>
